README.rst

```diff
@@ -1452,9 +1452,20 @@ Choose from the list of available rules:
     ``'newest'``
   - ``use_class_const`` (``bool``): use ::class notation; defaults to ``true``
 
-* **php_unit_ordered_covers** [@PhpCsFixer]
+* **php_unit_ordered_annotations** [@PhpCsFixer]
 
-  Order ``@covers`` annotation of PHPUnit tests.
+  Order annotations of PHPUnit tests.
+
+  Configuration options:
+
+  - ``annotations`` (a subset of ``['author', 'covers', 'coversNothing',
```

Comment:
Just thinking, since this fixer handles annotations which are not bound to `phpunit` test cases, such as

- `@author`
- `@internal`

and since we already have `PhpdocOrderFixer`, perhaps this fixer here should be renamed to `PhpdocOrderByValueFixer`?

Here's what I think:

### Current Fixers

- `PhpDocOrderFixer` groups annotations by name (`@param`, `@throws`, `@return`) and inserts an empty line between them

### New Fixers

- `PhpDocOrderByValueFixer` sorts configured annotations by value
- `PhpDocOrderByNameFixer` sorts configured annotations by name, and optionally inserts a blank line between them

### Deprecated Fixers

* `PhpDocOrderFixer` can be deprecated and proxy to the `PhpDocOrderByNameFixer`

Reply:
@julienfalque @keradus @kubawerlos @SpacePossum 

What do you think?

Reply:
That's exactly what I was thinking when reading the changes, 👍 for me. Though I think we can merge this PR (after renaming the new fixer) and work on the other fixers in another PR?

Reply:
@julienfalque 

Sounds good!

Renamed the fixer!